folsom/src/main/java/com/spotify/folsom/client/DefaultRawMemcacheClient.java

```diff
@@ -112,7 +104,8 @@ public static CompletionStage<RawMemcacheClient> connect(
       final Metrics metrics,
       final int maxSetLength,
       final EventLoopGroup eventLoopGroup,
-      final Class<? extends Channel> channelClass) {
+      final Class<? extends Channel> channelClass,
+      final SSLEngineFactory sslEngineFactory) {
```

Comment:
I strongly recommend overloading here, even if the original method just calls this new one with an extra `null` in the end. This is because this change without overloading would force library users to change their code to add an extra parameter.

Reply:
I was following @protocol7 's recommendations when I removed an extra constructor. The point is that constructor is not a part of public API, also it's highly unlikely one would construct `DefaultRawMemcacheClient` manually, builder gives a lot of extra features like retries and Ketama support